src/V/index.mjs

```diff
@@ -1378,6 +1381,83 @@ const V = (function() {
         return xml;
     };
 
+    const _attributeNames = Object.create(null);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#null-prototype_objects

Do we need to use a null-prototype object here? It can apparently lead to unexpected behavior, which may be problematic (e.g. if a customer tries to modify the list of attributes for which not to split camel-case).

I think we should be able to use a normal object here, since we are then using a Proxy object anyways to specify the custom getter (line 1444).

Reply:
What problems specifically?

This:

```js
V('rect', { toString: true }).node
```

Should produce:

```svg
<rect to-string="true"/>
```

And it would not if we define `_attributeNames` as `{}`.

Reply:
I'm not questioning that part, but my point is that the `{}` which would be used as a replacement would be a normal Object (i.e. with an Object prototype) - so if using that is okay, then it should be okay to say `const _attributes = {}` on this line by default, as well.

Examples of potential problems of using null-prototype objects according to the MDN link:
- An object with a null prototype can behave in unexpected ways, because it doesn't inherit any object methods from Object.prototype. This is especially true when debugging, since common object-property converting/detecting utility functions may generate errors, or lose information (especially if using silent error-traps that ignore errors).
- The main problem problem is with missing `toString`, `valueOf`, `hasOwnProperty` functions and the `constructor` property

Reply:
Added a comment explaining why we use an object without prototype.
Note It's a private variable and can not be accessed from outside.